tools/livecd-iso-to-pxeboot.sh

```diff
@@ -93,9 +95,11 @@ mkdir tftpboot
 # initrd image.  The Linux kernel will do the right thing,
 # aggregating both cpio archives (initrd + ISO) into a single
 # filesystem.
-ISOBASENAME=`basename "$ISO"`
-ISODIRNAME=`dirname "$ISO"`
-( cd "$ISODIRNAME" && echo "$ISOBASENAME" | cpio -H newc --quiet -L -o ) |
+NEWISO=$STRIPPEDISO/`basename "$ISO"`
+mkisofs --quiet -J -joliet-long -r -T -o $NEWISO --root LiveOS $CDMNT/LiveOS/squashfs.img
```

Comment:
And here is the thing, this creates simple ISO with joliet (I guess that's not even necessary).